content/post/know-your-aws-community-day-india-2020-speaker-abesh-rajasekharan.md

```diff
@@ -0,0 +1,21 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T16:54:08Z
+description = "Abesh Rajasekharan will be speaking on \"Using Ansible to manage Hybrid Architecture\" at <time> on <date> at AWS Community Day India 2020."
```

Comment:
placeholders should be updated/removed

Reply:
Corrected.

Reply:
@dA505819 the description hasn't been updated

Reply:
@SathyaBhat Already corrected. Checked twice on forestry.io.